pkg/cmd/init/exec.go

```diff
@@ -353,3 +359,27 @@ func (o *Options) deploySingletonControlplane(kubeClient kubernetes.Interface) e
 	}
 	return nil
 }
+
+func getRegistrationDrivers(o *Options) ([]operatorv1.RegistrationDriverHub, error) {
```

Comment:
could we make it simpler? we should validate and return error when a driver name is not allowed in advance.

```go
for _, driver := range o.registrationAuth:
    registrationDriver :=  operatorv1.RegistrationDriverHub{AuthType: driver}
   if driver == "awisra" {
      ...
   }
   registrationDrivers = append(registrationDrivers, registrationDriver)
```

Reply:
Done